sdk/test/UnitTests/Custom/Runtime/EndpointResolverTests.cs

```diff
@@ -19,15 +19,16 @@ public class EndpointResolverTests
         private const string AwsStsRegionalEndpointsEnvironmentVariable = "AWS_STS_REGIONAL_ENDPOINTS";
         private const string DefaultStsEndpoint = @"https://sts.amazonaws.com/";
 
-        [TestMethod][TestCategory("UnitTest")]
+        [TestMethod]
+        [TestCategory("UnitTest")]
         [TestCategory("Runtime")]
         public void TestSuccessfulCall()
         {
             var endpointResolver = new EndpointResolver();            
             var executionContext = CreateExecutionContext(SetupConfig());
```

Comment:
The `config` returned by the setup method now uses the `Standard` mode, so the determined endpoints have changed.